tests/mongo-go-driver.yml

```
@@ -12,48 +12,8 @@ args:
 # - ./mongo/integration/unified
 
 results:
-  ferretdb:
-    stats:
-      unexpected_fail: 0
-      unexpected_skip: 0
-      unexpected_pass: 0
-      expected_fail: 22
-      expected_skip: 17
-      expected_pass: 675
-
-    pass:
-      - go.mongodb.org/mongo-driver/mongo/TestClient/
-
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/DeleteExamples
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/IndexExamples
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/InsertExamples
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/QueryEmbeddedDocumentsExamples
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/QueryNullMissingFieldsExamples
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/QueryToplevelFieldsExamples
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/RunCommandExamples
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/StableAPExamples
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/TestDocumentationExamples/DeleteExamples
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples/TestDocumentationExamples/QueryNullMissingFieldsExamples
-
-    fail:
-      # database/collection names https://github.com/FerretDB/FerretDB/issues/125
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestAggregationExamples
-      - go.mongodb.org/mongo-driver/examples/documentation_examples/TestDocumentationExamples
-
-      # projection https://github.com/FerretDB/FerretDB/issues/80
-      - go.mongodb.org/mongo-driver/mongo/gridfs/TestGridFS
-
-      # aggregate https://github.com/FerretDB/FerretDB/issues/9
-      - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestAggregate
-      - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestAggregate/AllowDiskUse
-      - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestAggregate/Multiple_Batches
-      - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestAggregate/TestMaxTimeMSInGetMore
-      - go.mongodb.org/mongo-driver/mongo/TestClient/endSessions/number_of_sessions_divides_evenly
-      - go.mongodb.org/mongo-driver/mongo/TestClient/endSessions/number_of_sessions_does_not_divide_evenly
-      - go.mongodb.org/mongo-driver/mongo/TestClient/endSessions
-      - go.mongodb.org/mongo-driver/mongo/TestClient
-
-    skip: # shared with mongodb
+  common: # shared between ferretdb and mongodb
```

```suggestion
  common:
```

That comment was there to indicate that this section should be synced manually, but that's no longer need, and the name "common" is clear enough